protocol/messenger_contacts.go

```diff
@@ -16,6 +16,55 @@ import (
 	"github.com/status-im/status-go/protocol/transport"
 )
 
+func (m *Messenger) prepareMutualStateUpdateMessage(contactID string, updateType MutualStateUpdateType, clock uint64, timestamp uint64, outgoing bool) (*common.Message, error) {
+	var text string
+
+	var to string
+	var from string
+	if outgoing {
+		to = contactID
+		from = m.myHexIdentity()
+
+		switch updateType {
+		case MutualStateUpdateTypeSent:
+			text = "You sent a contact request to @" + to
```

Comment:
A broader question, maybe @cammellos can clarify. Why does the server need to be concerned with the presentation layer, I mean the text? Aren't clients better equipped to do this job?

Reply:
Yes, that's correct, we had a discussion with @MishkaRogachev and the agreement is that it's better to do that on the client side. 
Having said that, is a bit more complex and we already have a pattern that generates the message text here in status-go (it was done before when we had a command line client), and for the sake of simplicity, we agreed that's ok for now, since translations are not in scope for mvp (once we do translations, we'd want to change this behavior).
Do you think that's ok, or you'd rather address in this PR?

Reply:
> You mean translations are out of scope for status-go during the MVP? Because in the clients we already handle this quite decently.

Yes, we have everything in places since we had that from before, but we won't be supporting it yet


Makes sense to have decisions records and/or creating issues, we had that in status-mobile before, maybe it's something that would be handy for status-go as well

Reply:
sorry for the late reply, we used this in the past https://github.com/status-im/status-mobile/blob/develop/doc/decisions/0001-record-decisions.md